ui/app/AppLayouts/Chat/views/private/MembersSelectorBase.qml

```diff
@@ -31,28 +32,33 @@ InlineSelectorPanel {
     suggestionsModel: SortFilterProxyModel {
         id: suggestionsModel
 
-        function searchPredicate(displayName, localNickname, nameAlias) {
-            return displayName.toLowerCase().includes(root.edit.text.toLowerCase()) ||
-                   localNickname.toLowerCase().includes(root.edit.text.toLowerCase()) ||
-                   (!displayName && nameAlias.toLowerCase().includes(root.edit.text.toLowerCase()))
-        }
-
         function notAMemberPredicate(pubKey) {
-            for(var i = 0; i < model.count; i++) {
+            for (var i = 0; i < model.count; i++) {
```

Comment:
```suggestion
            for (let i = 0; i < model.count; i++) {
```
there is no room for `var` in modern js ;)

Reply:
Ah right, I missed it. It's old code :smile: